src/main/java/com/michelin/ns4kafka/services/ResourceQuotaService.java

```diff
@@ -127,6 +127,24 @@ public List<String> validateNewResourceQuota(Namespace namespace, ResourceQuota
             }
         }
 
+        String producerByteRate = resourceQuota.getSpec().get(USER_PRODUCER_BYTE_RATE.getKey());
```

Comment:
Could you add a unit test in `ResourceQuotaServiceTest` for these new conditions ?

Reply:
Done